Source/WebCore/dom/Element.cpp

```diff
@@ -2059,7 +2059,8 @@ static inline AtomString makeIdForStyleResolution(const AtomString& value, bool
 bool Element::isElementReflectionAttribute(const Settings& settings, const QualifiedName& name)
 {
     return (settings.ariaReflectionForElementReferencesEnabled() && name == HTMLNames::aria_activedescendantAttr)
-        || (settings.popoverAttributeEnabled() && name == HTMLNames::popovertargetAttr);
+        || (settings.popoverAttributeEnabled() && name == HTMLNames::popovertargetAttr)
+        || (settings.invokeAttributeEnabled() && (name == HTMLNames::invoketargetAttr || name == HTMLNames::invokeactionAttr));
```

Comment:
`HTMLNames::invokeactionAttr` should be removed, it doesn't reflect to an `Element` property.